python/src/opendp/measurements.py

```diff
@@ -202,10 +211,15 @@ def make_base_discrete_gaussian(
 def then_base_discrete_gaussian(
     scale,
     MO: Optional[RuntimeTypeDescriptor] = "ZeroConcentratedDivergence<QO>"
-):
-    r"""Partial constructructor which uses the preceding output domain and metric as its input domain and metric. See also:
+):  
+    r"""partial constructor of make_base_discrete_gaussian
+
+    .. seealso:: 
+      Delays application of `input_domain` and `input_metric` in :py:func:`opendp.measurements.make_base_discrete_gaussian`
 
-    :py:func:`opendp.measurements.make_base_discrete_gaussian`
+    :param scale: Noise scale parameter for the gaussian distribution. `scale` == standard_deviation.
+    :param MO: Output measure. The only valid measure is `ZeroConcentratedDivergence<QO>`, but QO can be any float.
```

Comment:
- If this is processed as RST, do you want double backticks here?
- "but QO can be any float" confuses me. `QO` is a type, right? Something like `ZeroConcentratedDivergence<1.5>` doesn't make sense.
- rephrase, unless there is actually `standard_deviation` parameter?
```suggestion
    :param scale: Noise scale parameter for the gaussian distribution. Equal to the standard deviation.
    :param MO: Output measure. The only valid measure is `ZeroConcentratedDivergence<QO>`, but QO can be any float.
```